src/components/project/project.service.ts

```diff
@@ -1200,4 +1243,25 @@ export class ProjectService {
       ).every((n) => n)
     );
   }
+
+  protected async validateOtherResourceId(
+    ids: string[],
+    label: string,
+    resourceField: string,
+    errMsg: string
+  ): Promise<boolean> {
```

Comment:
We are just throwing an exception if needed, which is the best way. The boolean return does nothing. We don't want to suggest to callers of this that they need to check that boolean to determine success.
```suggestion
  ): Promise<void> {
```